src/objects/zcl_abapgit_object_iobj.clas.abap

```diff
@@ -0,0 +1,170 @@
+CLASS zcl_abapgit_object_iobj DEFINITION PUBLIC INHERITING FROM zcl_abapgit_objects_super FINAL.
+
+  PUBLIC SECTION.
+    INTERFACES zif_abapgit_object.
+    ALIASES mo_files FOR zif_abapgit_object~mo_files.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+ENDCLASS.
+
+
+
+CLASS zcl_abapgit_object_iobj IMPLEMENTATION.
+
+
+  METHOD zif_abapgit_object~changed_by.
+
+    DATA: ls_viobj TYPE rsd_s_viobj,
+          lv_objna TYPE rsd_iobjnm.
+
+    lv_objna = ms_item-obj_name.
+
+    CALL FUNCTION 'RSD_IOBJ_GET'
+      EXPORTING
+        i_iobjnm  = lv_objna
+        i_objvers = 'A'
+      IMPORTING
+        e_s_viobj = ls_viobj.
+
+    rv_user = ls_viobj-tstpnm.
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~delete.
+
+    DATA: lt_iobjname TYPE rsd_t_c30.
+
+    APPEND ms_item-obj_name TO lt_iobjname.
+
+    CALL FUNCTION 'RSDG_IOBJ_MULTI_DELETE'
+      EXPORTING
+        i_t_iobjnm = lt_iobjname.
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~deserialize.
+
+    DATA: ls_details TYPE bapi6108,
+          lt_infoobj TYPE STANDARD TABLE OF bapi6108io.
+
+    io_xml->read( EXPORTING iv_name = 'IOBJ'
+                CHANGING cg_data = ls_details ).
+
+    CALL FUNCTION 'BAPI_IOBJ_CREATE'
+      EXPORTING
+        details = ls_details.
```

Comment:
What if create isn't successful? Maybe we should evaluate `return`?

Reply:
You are right. Evaluation of return added.